### DISCUSSION TITLE
custom titles for events uploaded by the CrashReporter

### DISCUSSION THREAD

**Discussion by alex-fennell-mtntp** [getsentry/sentry-unreal#628] [Q&A]
Hi there,

We end up with a ton of issues that are uploaded from fatal logging statements generated by our UE4 client that all end up displaying FOutputDevice::Logf for the main title which I assuming is being pulled from the callstack. Is there any way to assign a Json field to the __sentry data section that allows you override that default behavior? For example have it display Fatal Error: LowLevelFatalError [File:my_actor.cpp] [Line: 47] This is my fatal log message

**Comment by tustanivsky** [comment]
Hey @alex-fennell-mtntp, just want to clarify - do you use Unreal's crash reporter or native integration that the plugin offers?

**Comment by alex-fennell-mtntp** [comment]
We are using both. Currently we are sending a variety of events via the plugin but fatal errors are using the crash reporter. I was hoping their was an easy way to just add some metadata and everything would work - I am assuming that changing over to using the plugin would be better long term?

**Reply by tustanivsky** [reply]
Currently, there's no way to override how the events captured by Crash Reporter are processed on the server side. However switching to native integration should give you more flexibility in terms of configuring things and allow doing basic events pre-processing before uploading them to Sentry, so if you're on UE 5.2 or newer it's worth giving a try.

**Comment by alex-fennell-mtntp** [comment]
If I want to send a crash via SubsystemNativeImpl->CaptureException rather than by using to Unreal Crash Reporter, is there any way to upload the minidump? The documentation doesn't really cover this situation.

**Reply by tustanivsky** [reply]
`SubsystemNativeImpl->CaptureException` is an internal API used for handling specific cases (i.e. assertion/unsure capturing). Crashes are intercepted by `crashpad` if native integration is enabled and it should upload minidump automatically.
